packages/cdk8s-cli/bin/cmds/import.ts

```diff
@@ -13,24 +13,50 @@ class Command implements yargs.CommandModule {
   public readonly aliases = [ 'gen', 'import', 'generate' ];
 
   public readonly builder = (args: yargs.Argv) => args
-    .positional('SOURCE', { default: config.imports, desc: `The API source to import (supported sources: k8s, crd.yaml, https://domain/crd.yaml)`, array: true })
-    .example(`cdk8s import k8s`, `Imports Kubenretes API objects to imports/k8s.ts`)
-    .example(`cdk8s import k8s@1.13.0`, `imports a specific version of the Kubenretes API`)
-    .example(`cdk8s import jenkins.io_jenkins_crd.yaml`, 'imports constructs for the Jenkins custom resource definition from a file')
+    .positional('SPEC', { default: config.imports, desc: `import spec with the syntax [NAME=]SOURCE where NAME is an optional module name and supported SOURCEs are: k8s, crd.yaml, https://domain/crd.yaml)`, array: true })
```

Comment:
Thinking the config files should have a list of `importSpecs` instead of `imports`. Thoughts?

Reply:
leave as `imports`

Reply:
Sounds good to me!